bower.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "transportapi.js",
-  "version": "0.0.7",
+  "version": "0.1.0",
   "main": "dist/scripts/transportapi.min.js",
   "description": "A proxy API to TAPI endpoints",
-  "license": "TODO",
+  "license": "",
```

Comment:
No apparently that field in the bower descriptor [should be](https://github.com/bower/spec/blob/master/json.md#license) an "SPDX license identifier". So should be just `"Unlicense"`

...however we could put that text in the README